src/terra-dev-site/guide/Internationalization/AddingTranslations.b.guide.md

```diff
@@ -0,0 +1,13 @@
+# Adding Translations To Components
+
+When adding translations to a component, we recommend putting the translations in a directory named `translations`. The aggregate translations tool will look for translations in [the following directories](https://github.com/cerner/terra-toolkit/blob/master/scripts/aggregate-translations/defaultSearchPatterns.js) by default.
```

Comment:
should we specify here that translations goes a maximum of three levels deep?

Reply:
Updated: https://github.com/cerner/terra-ui/pull/132/commits/576255a658865bd7bfbc7e454abbbce66e51b2d2